webhook/resources/node/validator.go

```diff
@@ -73,7 +73,7 @@ func (n *nodeValidator) Update(request *admission.Request, oldObj runtime.Object
 
 	// Ensure the node controller already syncs the disk spec and status.
 	if !isNodeDiskSpecAndStatusSynced(oldNode) {
-		return werror.NewConflict(fmt.Sprintf("spec and status of disks on node %v are being syncing and please retry later.", oldNode.Name))
+		return werror.NewInvalidError(fmt.Sprintf("spec and status of disks on node %v are being syncing and please retry later.", oldNode.Name), "")
```

Comment:
Can we use `StatusReasonForbidden` instead? because InvalidError means invalid payload provided by the caller, but here the case is actually caused by the delayed disk info update causing some mismatches.

Reply:
Yes, we can. Sounds good.